multi_test_engine/src/MultiTestEngine.php

```diff
@@ -10,20 +10,60 @@ public function run() {
     $results = array();
 
     foreach ($engines as $engine_or_configuration) {
+      $include = false;
+      $include_report = false;
+      $changedFileMatchesInclude = false;
+
       if (is_array($engine_or_configuration)) {
         $engine_class = $engine_or_configuration['engine'];
-
         foreach ($engine_or_configuration as $configuration => $value) {
-          if ($configuration != 'engine') {
+          if ($configuration != 'engine' &&
+              $configuration != 'include' &&
+              $configuration != 'include_report') {
             $config->setRuntimeConfig($configuration, $value);
           }
+
+          if ($configuration == 'include' || $configuration == 'include_report') {
+            $$configuration = $value;
```

Comment:
Ouch, these variable-variables name looks so creepy. I know it is a bit more verbose, but wouldn't you mind to use change it to something like:

```php
switch($configuration) {
  case 'include': 
    $include_files = $value;
    break;
  case 'include_report':
    $include_report = $value
    break
}
```